views/js/previewer/plugins/tools/scale/component/scss/devicesPreviewer.scss

```diff
@@ -13,7 +13,7 @@
     $previewFrameShadow: rgba(0, 0, 0, .7);
 
     width: 100%;
-    height: 100%;
+    padding-top: 3rem;
```

Comment:
why do we need to touch the scale plugin? 🤔

Reply:
Unfortunately I have scroll bars for .devices-previewer - `height: 100%;` when the highlighter element is added to .content-wrapper :pensive:

Reply:
Can we add  `   display: flex;
    flex-direction: column;
` to .content-wrapper or it will break something, WDYT?

Reply:
Well, this is hard to tell. AFAIR the `devicesPreviewer` plugin was a pain to make working on every browsers.

Reply:
Seems like `.item-previewer-scope -> .content-wrapper` is used only here
https://github.com/oat-sa/extension-tao-testqti-previewer/blob/c9fa1ab57879035312574c8e55c0ae219ca03975/views/js/previewer/provider/item/scss/inc/_previewer-layout.scss#L80